server/graphql/types/src/types/draft_outbound_shipment_line.rs

```diff
@@ -0,0 +1,93 @@
+use async_graphql::{dataloader::DataLoader, *};
+use chrono::NaiveDate;
+use graphql_core::{loader::LocationByIdLoader, ContextExt};
+use service::invoice_line::get_draft_outbound_lines::DraftOutboundShipmentLine;
+
+use super::LocationNode;
+
+pub struct DraftOutboundShipmentItemData {
+    pub lines: Vec<DraftOutboundShipmentLine>,
+    pub placeholder_quantity: Option<f64>,
+}
+
+#[Object]
+impl DraftOutboundShipmentItemData {
+    pub async fn draft_lines(&self) -> Vec<DraftOutboundShipmentLineNode> {
+        DraftOutboundShipmentLineNode::from_vec(self.lines.clone())
+    }
+
+    pub async fn placeholder_quantity(&self) -> Option<f64> {
+        self.placeholder_quantity
+    }
```

Comment:
Returning placeholder quan separately - draft lines is therefore just the StockOut lines. easier on logic that constantly filtering for a placeholder...